test/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests.csproj

```diff
@@ -1,10 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>net7.0;net6.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
It is against #1031.

As mentioned in previous comment. Together with renaming package it is good opportunity to drop support for .NET Core 3.1. -> just remove netstandard2.0 version and make apriopriate changelog statement.

Reply:
I can remove this change from the PR. Would you like me to also remove the `netstandard2.0` target?

Reply:
I would say - yes, but the final decision is on @srprash